e2e/e2e-values.yml

```diff
@@ -38,25 +38,38 @@ scrapersConfig:
         - namespace: kube-system
           selector: "tier=control-plane,component=etcd"
           matchNode: true
-          url: http://localhost:2381
+          endpoints:
+            - url: http://localhost:2381
```

Comment:
this is what I meant, checking the nil pointer no need for `type` so specify no-auth :)

Reply:
actually, now that I think about it, you had the same idea